CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## Fixed
--   Fix windows build
+-   Fix windows build.
+-   Do not panic if lnd is not available during a halight swap execution.
```

Comment:
Can we rephrase this so it makes sense for the user? I think we should be talking about certificate and macaroon files here and what it "panic" means if you want to use that word. For example, only the tokio worker thread would have panicked which is something the user doesn't even see.

The only observable behaviour is that the swap doesn't do anything.